src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,14 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
 /**
  * <p>Дано файл, потрібно прочитати його вміст і вибрати всі слова що починаються на `w`.
  * Результат повернути у вигляді відсортованого масиву (за замовчуванням). Всі слова повинні
  * бути в нижньому регістрі. У випадку якщо таких слів не знайдено повернути пустий масив.
  * Приклад: Width world Wide web
  * Результат: web wide width world</p>
  */
+
 public class FileWork {
+    static final String LETTER = "w";
+    static final String POINT = ".";
+    static final String EMPTY = "";
+    static final String SPACE = " ";
+    static final String EXCLAMATION_MARK = "!";
+    static final String QUESTION_MARK = "?";
```

Comment:
1) add access modifiers to your constants;
2) Do you need all of them?

Reply:
done